src/Symfony/Component/HttpKernel/Controller/ArgumentValueResolver/VariadicRequestAttributeResolver.php

```diff
@@ -18,25 +18,27 @@
 /**
  * Grabs the variadic value from the request and returns it.
  *
- * Opposite of {@see ArgumentFromAttribute}.
+ * Opposite of {@see ArgumentFromAttributeResolver}.
```

Comment:
I don't understand why the two are "opposite" and how this is relevant. Can you elaborate? Perhaps we can even remove this comment.